es6/spec/database/query/equalTo.spec.js

```diff
@@ -80,6 +85,30 @@ describe(".equalTo(query)", () => {
     queryA.equalTo(queryB).should.be.false;
   });
 
+  describe("(where)", () => {
+    it("should treat where equals to andWhere", done => {
+      const name = "aName";
+
+      database.mock.select(/.*/).from(/u.*rs/).where("created_at", ">", /.*/).where("name", name).results(mockUsers);
```

Comment:
If we're just testing .where, we don't really want to have all of this complicated regex. If regex isn't necessary to exemplify; simplify instead.